src/components/ButtonBase/ButtonBase.style.ts

```diff
@@ -66,7 +65,7 @@ export const buttonBaseStyle = ({
       );
     }
 
-    return pickTextColorFromSwatches(calculatedColor.color, calculatedColor.shade);
+    return theme.utils.getAAColorFromSwatches(calculatedColor.color, calculatedColor.shade);
```

Comment:
Can't you directly import `getAAColorFromSwatches`? There're many usages of it and having the `theme.utils` each time seems kinda redundant (?)

Reply:
We can of course , but it's about being consistent in the way we use the theme utils. It's like importing the `getColor` function. We can do it, but we always prefer to use it through the theme.